test/controllers/endorseCompanyClosureCertificate.controller.test.ts

```diff
@@ -56,4 +68,48 @@ describe('EndorseCompanyClosureCertificateController', () => {
       assert.isTrue(htmlAssertHelper.hasText('span#applicantName', 'John Smith'))
     })
   })
+
+  describe('POST - ensure form submission is handled correctly', () => {
+    it('should redirect successfully if validator returns no errors', async () => {
+      const testObject: EndorseCertificateFormModel = {confirmation: 'understood'}
+      const dissolutionPatchResponse: DissolutionPatchResponse = {
+        links: {
+          self: '',
+          payment: ''
+        }
+      }
```

Comment:
There should be a fixture called generateDissolutionPatchResponse() that should be used here instead. In fact, I don't think the controller needs to know about DissolutionPatchResponse at all. We don't use any values from it.